pattern-1/marathon-applications/wso2apim.json

```diff
@@ -0,0 +1,107 @@
+  "cpus": 1,
+  "mem": 2048,
+  "disk": 0,
+  "instances": 1,
+  "acceptedResourceRoles": [
+    "*"
+  ],
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "forcePullImage": false,
+      "image": "wso2am-dcos:2.1.0",
+      "parameters": [],
+      "privileged": false
+    },
+    "volumes": [
+      {
+        "containerPath": "/home/wso2carbon/wso2am-analytics-2.1.0/repository/conf/carbon.xml",
+        "hostPath": "/volumes/wso2/apim-analytics/repository/conf/carbon.xml",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/home/wso2carbon/wso2am-analytics-2.1.0/repository/conf/api-manager.xml",
+        "hostPath": "/volumes/wso2/apim-analytics/repository/conf/api-manager.xml",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/home/wso2carbon/wso2am-analytics-2.1.0/repository/conf/datasources/master-datasources.xml",
+        "hostPath": "/volumes/wso2/apim-analytics/repository/conf/datasources/master-datasources.xml",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/home/wso2carbon/wso2am-2.1.0/repository/components/lib/mysql-connector-java-5.1.34-bin.jar",
+        "hostPath": "/volumes/wso2/apim/repository/components/lib/mysql-connector-java-5.1.34-bin.jar",
+        "mode": "RW"
+      }
+    ],
+    "portMappings": [
+      {
+        "containerPort": 9763,
+        "hostPort": 0,
+        "labels": {
+          "VIP_0": "/wso2/apim:9763"
+        },
+        "name": "servlet-http",
+        "protocol": "tcp",
+        "servicePort": 9763
+      },
+      {
+        "containerPort": 9443,
+        "hostPort": 0,
+        "labels": {
+          "VIP_0": "/wso2/apim:9443"
+        },
+        "name": "servlet-https",
+        "protocol": "tcp",
+        "servicePort": 9443
+      },
+      {
+        "containerPort": 8280,
+        "hostPort": 0,
+        "labels": {
+          "VIP_0": "/wso2/apim:8280"
+        },
+        "name": "passthrough-http",
+        "protocol": "tcp",
+        "servicePort": 8280
+      },
+      {
+        "containerPort": 8243,
+        "hostPort": 0,
+        "labels": {
+          "VIP_0": "/wso2/apim:8243"
+        },
+        "name": "passthrough-https",
+        "protocol": "tcp",
+        "servicePort": 8243
+      }
+    ]
+  },
+  "healthChecks": [
+    {
+      "gracePeriodSeconds": 300,
+      "intervalSeconds": 60,
+      "maxConsecutiveFailures": 3,
+      "portIndex": 0,
+      "protocol": "TCP",
+      "timeoutSeconds": 20,
+      "delaySeconds": 15
+    }
+  ],
+  "labels": {
+    "HAPROXY_GROUP": "external",
+    "HAPROXY_1_VHOST": "wso2apim",
+    "HAPROXY_1_BACKEND_SERVER_OPTIONS": "server {serverName} {host_ipv4}:{port} {cookieOptions} check inter 5s ssl verify none",
+    "HAPROXY_3_VHOST": "wso2apim-gateway",
+    "HAPROXY_3_BACKEND_SERVER_OPTIONS": "server {serverName} {host_ipv4}:{port} {cookieOptions} check inter 5s ssl verify none"
+  },
+  "networks": [
+    {
+      "mode": "container/bridge"
+    }
+  ]
+}
```

Comment:
Shall we keep new line here to avoid this RED symbol ?

Reply:
+1 Thanks Dilan! Will add a blank line here.

Reply:
This is fixed in commit https://github.com/imesh/dcos-apim/commit/fe8ce169d81a252667ca4447b4e0b4b5ca6bd595